### PR TITLE
Fix for #135 and #42

### PR DESCRIPTION
**Pull request by henryso**

### CODE REVIEW

File: lib/characters.c [hunk 1]
```diff
@@ -1114,6 +1108,5 @@ gregorio_rebuild_first_syllable (gregorio_character ** param_character)
   current_character = *param_character;
   gregorio_go_to_first_character (&current_character);
   (*param_character) = current_character;
-  // this is not working, so people have to mind the way they put text styles on the initial.
-  //gregorio_rebuild_characters (&current_character, CENTER_FULLY_DETERMINED);
 }
+/* vim: set ts=8 sts=2 sw=2 noet: */
```
Comment (eroux) [inline lib/characters.c]: I admit I'm a bit against having editor-specific tags in files... I like the idea of editorconfig, and there's [a plugin for vim](https://github.com/editorconfig/editorconfig-vim#readme). This allows to put preferences once for the whole project, and change them easily... What do you think?

**Comment by eroux** [comment]
Apart from my small comment (not really blocking), I think I'm ready to merge this PR (though I admit I don't understand each line of it, but I fully trust you). Maybe, before merging, there could be an associated test? This part of the code has proven to be a nest of bugs and things broke many times when there was a change in it, so a small non-regression test would be very useful I believe...

**Comment by henryso** [comment]
Sorry, I didn't mean to leave that in.  I'll scrap that line.

**Comment by henryso** [comment]
I will create tests for this in https://github.com/gregorio-project/gregorio-test

**Comment by eroux** [comment]
Ok perfect!

**Comment by eroux** [comment]
Does it address your last comment in #135 ?

**Comment by henryso** [comment]
A qualified yes. As you say, this code is a nest of bugs, so I'm trying to run it through as many tests as I can before I say for sure.

**Comment by eroux** [comment]
Just a reminder: can you please add this in the CHANGELOG?

**Pull request merged**